src/secret.rs

```diff
@@ -14,25 +14,23 @@
 
 //! Helpers for displaying secret values
 
-use ::core::fmt;
-use ::{SecretKey, KeyPair, to_hex};
-use ecdh::SharedSecret;
-use constants::SECRET_KEY_SIZE;
-
+use core::fmt;
+use crate::{to_hex, constants::SECRET_KEY_SIZE, key::{SecretKey, KeyPair}, ecdh::SharedSecret};
```

Comment:
This import statement is not uniform with the rest of the crate but lets ignore it for now and let the tooling sort it out later. I have various active efforts to resolve the `rustfmt` usage issue across the stack :)

Reply:
I'd love to use rustfmt for these kinds of things(with default configuration or not), although sadly it doesn't currently merge imports together. 
Either way, we should probably come up with some styling guide for rust-bitcoin, as currently I feel like different people prefer different things and then it's not consistent across the code base

Reply:
Fair point. AFAIU there is an unstable config option for this
`
imports_granularity = "Module"     # Default "Preserve"
```

I personally prefer the "Module" option because it makes, in my experience, for less noisy diffs as imports are added and removed. I'm trying to slowly resolve the rustfmt issue atm.

Reply:
> unstable config option

There is a useful[^1] bug in rustfmt where you can pass unstable config options via the commandline without needing nightly!

e.g. the following works with stable rustfmt (if you delete the rustfmt file that disables everything):

```shell
cargo fmt -- --config imports_granularity=module
```

You could piece together a rustfmt config using [aliases](https://doc.rust-lang.org/cargo/reference/config.html#configuration-format) that handpicks the options you want but also makes them easily accessible for CI and local development.

Editors won't pick that up unfortunately though.

[^1]: Useful because needing to install a nightly toolchain just for formatting is stupid IMO.

Reply:
Thanks for the tip, not working with an editor is not really acceptable IMO.